op-challenger/game/fault/types/position.go

```diff
@@ -1,56 +1,89 @@
 package types
 
-import "fmt"
+import (
+	"fmt"
+	"math/big"
+)
 
 // Position is a golang wrapper around the dispute game Position type.
 type Position struct {
 	depth        int
-	indexAtDepth int
+	indexAtDepth *big.Int
 }
 
-func NewPosition(depth, indexAtDepth int) Position {
-	return Position{depth, indexAtDepth}
+func NewPosition(depth int, indexAtDepth big.Int) Position {
```

Comment:
We should be use a pointer here to avoid shallow copies of `big.Int` which aren't supported.